docker-compose.yml

```diff
@@ -70,6 +70,7 @@ services:
       - "4566-4597:4566-4597"
       - 8999:8080
     environment:
+      - HOSTNAME_EXTERNAL=localstack
```

Comment:
do you need this?  I had to take it out as it was breaking my builds

Reply:
Maybe not...

Reply:
removed